conan/cli/command.py

```diff
@@ -25,6 +25,106 @@ def _fill_text(self, text, width, indent):
         text = textwrap.dedent(text)
         return ''.join(indent + line for line in text.splitlines(True))
 
+    def _format_usage(
```

Comment:
This is the implementation of the _format_usage method, but changing the order of optional and positional arguments

Reply:
I think this is too much, we most likely don't want this in our codebase.

I'd say we could just inject a line in the output help that says:

```
NOTE: Use always positional parameters first ``conan <command> <positional args> [optional args]``
```
Wdyt?